src/code.sh

```
@@ -1,29 +1,30 @@
-    mkdir input_files
-    find ~/in/input_file_array -type f -name "*" -print0 | xargs -0 -I {} mv {} ~/input_files
+    FILE_TEST_OUTPUT=$(file --brief "$flagstat_path")
+    if [[ $FILE_TEST_OUTPUT == "JSON data" ]]; then
+        READ_COUNT=$(jq -r '."QC-passed reads" | ."primary"' "$flagstat_path")
+    elif [[ $FILE_TEST_OUTPUT == "ASCII text" ]]; then
+        READ_COUNT=$(grep "primary$" "$flagstat_path" | awk '{print $1}')
+    fi
 
-    # or files can be downloaded one by one, specifying a name for them within the workstation
-    dx download "$input_file" -o input_file_name
+    TARGET_FRACTION=$(bc -l <<< "scale=3; $target_read_count / $READ_COUNT")
```

Shellcheck (0.11.0)

[warning] target_read_count is referenced but not assigned.

(SC2154)